Makefile

```diff
@@ -272,6 +272,23 @@ dev: prebuild-check deps generate $(FRESH_BIN)
 	docker-compose up -d db auth
 	F8_DEVELOPER_MODE_ENABLED=true $(FRESH_BIN)
 
+MINISHIFT_IP = `minishift ip`
+MINISHIFT_URL = http://$(MINISHIFT_IP)
+
+.PHONY: dev-wit-openshift
+dev-wit-openshift: prebuild-check deps generate $(FRESH_BIN)
+	eval `minishift oc-env` && oc new-project wit-openshift
+	AUTH_WIT_URL=$(MINISHIFT_URL):8080 kedge apply -f kedge/db.yml -f kedge/db-auth.yml -f kedge/auth.yml
+	F8_AUTH_URL=$(MINISHIFT_URL):31000 \
```

Comment:
Can we have a server/host-name which doesn't change? Because we'll have to keep it whitelisted.

Reply:
true, I had chat with @praveenkumar , but if I understand correctly there is no way as of now to set up static IP for minishift.
@gbraad @kadel  any clue?

Reply:
@sbose78 never mind. Saw https://github.com/fabric8-services/fabric8-wit/pull/1741#issuecomment-342107015

Reply:
Real Static-IP is only possible on Hyper-V due to how we can interact with the VM. For the other hypervisors we will always rely on first getting a DHCP-assigned address, however... I am trying to see how we can optimize this.

Reply:
thanks @LalatenduMohanty @gbraad for information, we are trying host entry in /var/hosts. I will update when done